scripts/router.js

```diff
@@ -0,0 +1,15 @@
+//Manipulate Pages SPA-Style
+page.base('');
+
+page('/', about);
+page('/projects', projects)
+
+page();
+
+function about() {
```

Comment:
hmmm... is there a reason not to just make these a part of your page commands? replace line 4 with:
```page('/', aboutController.init());```
?